scopes/compilation/compiler/types.ts

```diff
@@ -96,6 +96,11 @@ export interface Compiler extends CompilerOptions {
    */
   build(context: BuildContext): Promise<BuiltTaskResult>;
 
+  /**
+   * return the dist dir of the compiled files
+   */
+  getDistDir(): string;
```

Comment:
please document - abs or relative path? and relative to what?

Reply:
done